### PR TITLE
#1486 -- DetailsList: Fixes clipped button inside Details List

### PR DESCRIPTION
**Pull request by aleksandrjPersonal**
#### Pull request checklist

- [ ] Addresses an existing issue: #1486
- [ ] Include a change request file if publishing 
- [ ] New feature, bugfix, or enhancement
  - [ ] Includes tests
- [ ] Documentation update

#### Description of changes
DetailsList: Fixes clipped button inside Details List
See screen shot

#### Focus areas to test

Details List buttons

### CODE REVIEW

**Comment by aleksandrjPersonal** [comment]
screenshot 18

File: packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.scss [hunk 1]
```diff
@@ -53,6 +53,10 @@ $unselectedHoverColor: $ms-color-neutralLighter;
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
+
+  & > button{
```
Comment (MLoughry) [inline packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.scss]: Nit: spacing between `button` and `{`

**Comment by cschleiden** [comment]
Is this the right fix? What happens if maybe I don't want buttons in my details list to take up a 100% width? Maybe I have a couple of `IconButton`s?

**Comment by aleksandrjPersonal** [comment]
@cschleiden Good point, now limiting buttons to max-width: 100%

**Comment by cschleiden** [comment]
I mean, should we add that to the DetailsList at all? What about images, input elements, etc. ? There is so much content that could be in the list, do we want to add special styling for all of these types?

**Pull request merged**